### ISSUE TITLE
Week 3 Understanding of Concepts

### ISSUE THREAD

**Issue by alizea-alvarez** [Watts-College/paf-510-template#56]
@AntJam-Howell  @nickmcmullen

I want to make sure I'm understanding the concepts for week three before I head into the assignment.

So, control variables, in general, are variables outside of the policy variable (PV) that we include in the model because they may impact our dependent variable (DV). Ultimately, we do this to make sure the PV's effect on our DV is precise and accurate.

The two kinds of control variables are correlated and uncorrelated variables. 

Correlated variables are variables that relate to both the PV and the DV. Including them can remove bias because it accounts for confounding variables that may be present, but it can increase the standard error of coefficient because it now accounts for some of the variation in the data that our PV also explains. Ultimately, this makes the impact our program has on the DV more accurate but less precise.

Uncorrelated variables are variables that don't relate to our PV but may relate to our DV. Including them doesn't affect bias but reduces the residual variance because it explains more of the variation in our data that the PV doesn't explain, which leads to a smaller standard error of coefficient. Ultimately, this makes the impact our program has on our DV more precise but less accurate, as bias could still be present.

Is this correct? Or am I missing something?

**Comment by AntJam-Howell** [comment]
Your understanding is mostly correct, but there are a few important
clarifications to note:

 1. **Correlated Variables:** These control variables are related to both
the policy variable (PV) and the dependent variable (DV). Including them
helps remove bias from the estimated effect of the PV on the DV by
controlling for potential confounding factors. While this can increase
standard errors (reducing precision), it makes the estimate more
**accurate** by addressing confounding influences.

 2. **Uncorrelated Variables:** These are not related to the PV but may
still be related to the DV. Including them can reduce residual variance,
which lowers the standard error of the estimate and increases
**precision**. However, they do not address bias related to confounding
factors between the PV and DV.


 In summary:

- Correlated control variables reduce bias, increasing **accuracy** but
potentially lowering precision.

- Uncorrelated control variables reduce residual variance, increasing
**precision** but may not address bias.

 The goal is to strike a balance between controlling for confounding
factors and maintaining precision.



--
Anthony Howell
Director, Center for Technology, Data, and Society,
Associate Professor, Public Policy
Arizona State University

**Comment by alizea-alvarez** [comment]
@nickmcmullen @AntJam-Howell 

For Part I:
So, by controlling for another variable and plotting Class Size against the residuals, we're looking at what's left of that relationship after removing the influence of the control variable- which we can use to determine how much of the relationship between class size and test score was due to the control variable. 

We can interpret the change in relationship by the slope and the spread of the points. If the slope is flatter or the data points are more clustered, then the relationship between class size and test scores is weaker, and if it's relatively the same, then the relationship between class size and test scores remains the same. 

For Part II:
Correlation affects the slope and the standard error of the identified coefficient estimate. 


Is that right?
